services/dy-static-file-server/docker/custom/Dockerfile

```diff
@@ -8,38 +8,51 @@ FROM nginx:1.21.0-alpine as production
 #
 
 ARG PYTHON_VERSION="3.8.10-r0"
-ARG INPUT_DIR="/www/inputs"
-ARG OUTPUT_DIR="/www/outputs"
 ARG WORKDIR="/workdir"
 
+ENV SC_BUILD_TARGET=production
+ENV SERVER_ROOT="/www"
+ENV INPUT_FOLDER="${SERVER_ROOT}/inputs"
+ENV OUTPUT_FOLDER="${SERVER_ROOT}/outputs"
+ENV SC_USER_ID=101
+ENV SERVER_PORT=8080
+ENV SERVER_LOG_LEVEL=debug
+
+# creating own project's user
+ENV SC_USER_ID 9004
+ENV SC_USER_NAME scudy
+RUN adduser -D -u ${SC_USER_ID} -s /bin/sh -h /home/${SC_USER_NAME} ${SC_USER_NAME}
+
+
 LABEL maintainer=GitHK
-COPY docker/custom/nginx.conf /etc/nginx/nginx.conf
-COPY docker/custom/boot.sh /boot.sh
 
 RUN apk add --update --no-cache \
     "python3=${PYTHON_VERSION}" \
-    py3-pip
+    py3-pip \
+    su-exec
```

Comment:
you might be interested in this article, regarding the best python docker images:
https://link.medium.com/Fbb12XECQjb